src/views/registration-js/attendees/fields/birth.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/birth.php
+ *
+ * @since   TBD
+ *
+ * @see     Tribe__Tickets_Plus__Meta__Field__Birth
+ */
+
+$required    = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id   = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$birth_field = $field;
+$field       = (array) $field;
+$field_name  = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled    = false;
+$classes     = [
+	'tribe-common-b1',
+	'tribe-field',
+	'tribe-tickets__item__attendee__field__birth',
+	'tribe-tickets-meta-required' => $required,
+];
+?>
+<div class="tribe_horizontal_datepicker__container">
+	<div <?php tribe_classes( $classes ); ?> >
+		<label
+				class="tribe-common-b2--min-medium tribe-tickets-meta-label"
+				for="<?php echo esc_attr( $birth_field->month_id ); ?>"
+		><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+
+		<!-- Month -->
+		<div class="tribe_horizontal_datepicker">
+			<select
+					<?php tribe_disabled( $disabled ); ?>
+					<?php tribe_required( $required ); ?>
+					class="tribe_horizontal_datepicker__month"
+			>
+				<option value="" disabled selected><?php esc_html_e( 'Month', 'tribe-event-plus' ); ?></option>
+				<?php foreach ( $birth_field->get_months() as $month_number => $month_name ) : ?>
+					<option value="<?php echo esc_attr( $month_number ); ?>"><?php echo esc_html( $month_name ); ?></option>
```

Comment:
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and T_COLON is required

<sub>hash: 52cdee8bb5b09789cd543e1aee977908</sub>